addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1487,15 +1491,38 @@ options.registry.topMenuColor = options.Class.extend({
     /**
      * @override
      */
-    selectStyle(previewMode, widgetValue, params) {
-        this._super(...arguments);
-        const className = widgetValue ? (params.colorPrefix + widgetValue) : '';
+    async selectStyle(previewMode, widgetValue, params) {
+        await this._super(...arguments);
+        let toggleColorOption = 'header_color';
+        if (ColorpickerDialog.isCSSColor(widgetValue)) {
+            toggleColorOption = 'header_inline_color';
+            widgetValue = `background-color:${widgetValue}`;
+        } else {
+            widgetValue = params.colorPrefix + widgetValue;
+        }
```

Comment:
Is not there a problem here? You removed the check to ensure that widgetValue contains something so my guess it that the reset button does not work anymore?

Also header_inline_color should not contain the whole inline style, only the color. Remove the 'background-color:'

Also should not you remove the header_color value when a header_inline_color is chosen and also the opposite?